ytmusicapi/parsers/podcasts.py

```diff
@@ -61,10 +61,14 @@ def from_runs(cls, description_runs: List[Dict]) -> "Description":
 def parse_base_header(header: Dict) -> Dict:
     """parse common left hand side (header) items of an episode or podcast page"""
     strapline = nav(header, ["straplineTextOne"])
+    try:
+        author_id = nav(strapline, ["runs", 0, *NAVIGATION_BROWSE_ID])
+    except KeyError:
+        author_id = ""
     return {
         "author": {
             "name": nav(strapline, [*RUN_TEXT]),
-            "id": nav(strapline, ["runs", 0, *NAVIGATION_BROWSE_ID]),
+            "id": author_id,
```

Comment:
You could simply set `nav(..., True)` on the original nav statement

Reply:
Ah, thanks! I'll make that change.